fatal error: sql.h: No such file or directory

when pip install pyodbc 
got error 
  #include <sql.h>
              ^~~~~~~
    compilation terminated.
    error: command 'gcc' failed with exit status 1
    
    ----------------------------------------
Command "/usr/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-e89qqq9z/pyodbc/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-_itwd76b/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /tmp/pip-install-e89qqq9z/pyodbc/

This is the error below 
 " In file included from src/buffer.cpp:12:
    src/pyodbc.h:56:10: fatal error: sql.h: No such file or directory
     #include <sql.h>
              ^~~~~~~
    compilation terminated.
    error: command 'gcc' failed with exit status 1"

I got the following error when I build my golang project on centos7
# github.com/alexbrainman/odbc/api
../../pkg/mod/github.com/alexbrainman/odbc@v0.0.0-20200426075526-f0492dfa1575/api/api_unix.go:14:18: fatal error: sql.h: No such file or directory
 // #include <sql.h>
                  ^
compilation terminated.
make: *** [linux] Error 2

and just execute the following command to fix it
sudo yum install unixODBC-devel

--- a/.github/workflows/ubuntu_build.yml
+++ b/.github/workflows/ubuntu_build.yml
@@ -191,3 +191,23 @@ jobs:
       run: |
         cd "$GITHUB_WORKSPACE"
         python "./${{ matrix.tests-dir }}/mysqltests.py" "DRIVER={MySQL ODBC 8.0 ANSI Driver};SERVER=localhost;UID=root;PWD=root;DATABASE=test;CHARSET=utf8mb4"
+
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, windows-2019, macos-10.15]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build wheels
+        uses: pypa/cibuildwheel@v2.1.3
+        # to supply options, put them in 'env', like:
+        # env:
+        #   CIBW_SOME_OPTION: value
+
+      - uses: actions/upload-artifact@v2
+        with:
+          path: ./wheelhouse/*.whl

--- a/.github/workflows/ubuntu_build.yml
+++ b/.github/workflows/ubuntu_build.yml
@@ -1,6 +1,6 @@
 name: Ubuntu build
 
-on: [push]
+on: [push, pull_request]
 
 jobs:
   run_tests:

--- a/.github/workflows/ubuntu_build.yml
+++ b/.github/workflows/ubuntu_build.yml
@@ -206,7 +206,7 @@ jobs:
         uses: pypa/cibuildwheel@v2.1.3
         env:
           CIBW_MANYLINUX_X86_64_IMAGE: manylinux_2_24
-          CIBW_BEFORE_ALL_LINUX: apt-get -y install unixodbc-dev
+          CIBW_BEFORE_ALL_LINUX: apt-get update && apt-get -y install unixodbc-dev
 
       - uses: actions/upload-artifact@v2
         with:

--- a/.github/workflows/ubuntu_build.yml
+++ b/.github/workflows/ubuntu_build.yml
@@ -205,6 +205,7 @@ jobs:
       - name: Build wheels
         uses: pypa/cibuildwheel@v2.1.3
         env:
+          CIBW_MANYLINUX_X86_64_IMAGE: manylinux_2_24
           CIBW_BEFORE_ALL_LINUX: apt-get -y install unixodbc-dev
 
       - uses: actions/upload-artifact@v2

--- a/.github/workflows/ubuntu_build.yml
+++ b/.github/workflows/ubuntu_build.yml
@@ -207,6 +207,7 @@ jobs:
         env:
           CIBW_MANYLINUX_X86_64_IMAGE: manylinux_2_24
           CIBW_BEFORE_ALL_LINUX: apt-get update && apt-get -y install unixodbc-dev
+          CIBW_SKIP: "*-win32 *-manylinux_i686"
 
       - uses: actions/upload-artifact@v2
         with:

--- a/.github/workflows/ubuntu_build.yml
+++ b/.github/workflows/ubuntu_build.yml
@@ -206,6 +206,7 @@ jobs:
         uses: pypa/cibuildwheel@v2.1.3
         env:
           CIBW_MANYLINUX_X86_64_IMAGE: manylinux_2_24
+          CIBW_MANYLINUX_PYPY_X86_64_IMAGE: manylinux_2_24
           CIBW_BEFORE_ALL_LINUX: apt-get update && apt-get -y install unixodbc-dev
           CIBW_SKIP: "*-win32 *-manylinux_i686"
 

--- a/.github/workflows/ubuntu_build.yml
+++ b/.github/workflows/ubuntu_build.yml
@@ -204,9 +204,8 @@ jobs:
 
       - name: Build wheels
         uses: pypa/cibuildwheel@v2.1.3
-        # to supply options, put them in 'env', like:
-        # env:
-        #   CIBW_SOME_OPTION: value
+        env:
+          CIBW_BEFORE_ALL_LINUX: apt-get -y install unixodbc-dev
 
       - uses: actions/upload-artifact@v2
         with:

--- a/.github/workflows/ubuntu_build.yml
+++ b/.github/workflows/ubuntu_build.yml
@@ -206,9 +206,9 @@ jobs:
         uses: pypa/cibuildwheel@v2.1.3
         env:
           CIBW_MANYLINUX_X86_64_IMAGE: manylinux_2_24
-          CIBW_MANYLINUX_PYPY_X86_64_IMAGE: manylinux_2_24
           CIBW_BEFORE_ALL_LINUX: apt-get update && apt-get -y install unixodbc-dev
-          CIBW_SKIP: "*-win32 *-manylinux_i686"
+          # disable 32-bit and pypy builds
+          CIBW_SKIP: "*-win32 *-manylinux_i686 pp*"
 
       - uses: actions/upload-artifact@v2
         with:

--- a/setup.py
+++ b/setup.py
@@ -243,7 +243,7 @@ def get_version():
       1. If in a git repository, use the latest tag (git describe).
       2. If in an unzipped source directory (from setup.py sdist),
          read the version from the PKG-INFO file.
-      3. Use 4.0.0.0 and complain a lot.
+      3. Use 4.0.0.dev0 and complain a lot.
     """
     # My goal is to (1) provide accurate tags for official releases but (2) not have to manage tags for every test
     # release.
@@ -272,7 +272,7 @@ def get_version():
 
     if not numbers:
         _print('WARNING: Unable to determine version.  Using 4.0.0.0')
-        name, numbers = '4.0.0-unsupported', [4,0,0,0]
+        name, numbers = '4.0.0.dev0', [4,0,0,0]
 
     return name, numbers
 

--- a/setup.py
+++ b/setup.py
@@ -272,7 +272,7 @@ def get_version():
 
     if not numbers:
         _print('WARNING: Unable to determine version.  Using 4.0.0.0')
-        name, numbers = '4.0.0.dev0', [4,0,0,0]
+        name, numbers = '4.0.dev0', [4,0,0,0]
 
     return name, numbers
 

